app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -46,7 +46,7 @@ def prov_field_changed
         if refresh_divs
           all_dialogs.each do |dialog_name, dialog|
             if dialog[:display] == :show && dialog_name == @edit[:new][:current_tab_key]
-              page.replace_html(dialog_name, :partial => dialog_partial_for_workflow, :locals => {:wf => @edit[:wf], :dialog => dialog_name})
+              page.replace_html(dialog_name, :partial => dialog_partial_for_workflow, :locals => {:wf => @edit[:wf], :dialog => dialog_name, :canEdit => true})
```

Comment:
Why do we need both canEdit and isDisabled variables? isDisabled--> false doesn't mean canEdit is true?